cql/ManageSpecialPopulation.cql

```diff
@@ -55,6 +55,26 @@ context Patient
 // RECOMMENDATIONS FOR SPECIAL POPULATIONS
 //------------------------------------------------------------------------------
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+// Special Populations Management of Patients With Histologic Cancer
+
+define RecommendationForPatientsWithHistologicCancer:
+  case
+    when (
+      Rare.HistologyInterpretedAsCancer
+    ) then
+      {
+        short: 'Histologic Cancer',
+        date: Today(),
+        group: 'Special Population',
```

Comment:
The group here should be something like "Histologic Cancer". If you look at the other "group" identifiers in this file, they all denote which specific section they come from in the guidelines. This is what ends up displaying on the BPA and helps us to track down where a recommendation came from.

Reply:
Changed group to "Histologic Cancer"